### DISCUSSION TITLE
[Possible Bug]  More than hours before - workflow not triggering

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#14635] [How to] [answered]
I have situation where I have created basic workflow with condition "More than hours before". I set number 1 in it.
This workflow is not triggering.  I am asking here before reporting bug, as I am uncertain if this is reproducible in other instances.
Workflow created for custom module and the field used is "Created time"

Please help me verifying this state.

ps.
Also naming of workflows is kind of confusing, but unless I know how (and if)  it works I cannot discuss with it :-)

**Comment by waran70** [comment]
Coming up with support to this question. Trying to use fragments of hour in workflows:
- less than hours before
- than hours later
- more than hours before
- more than hours later

and it seems like not working. For instance I want to send email 15 minutes after record has been created - it seems to be basic requirement but there must be some blocker.

**Comment by JohnMaverick** [comment]
Following you gents I have tested I have tested **more than hours later** with condition "00:15:00" or 0:15 or 0.15 - not working. 
The condition is being ignored, and workflow is sent despite this condition is set (correctly or not).
Should we report bug?

**Comment by davide-alghi** [comment, marked as answer]
Hi all,
following conditions work fine

- less than hours before
- less than hours later
- more than hours before
- more than hours later

"Date smaller than now" and "Date greater than now" had some bugs, but they have been fixed and will be available in YF 6.2 - https://github.com/YetiForceCompany/YetiForceCRM/issues/14023

Below a graph to clarify the conditions behavior (value must be an integer)

![wfDateTimeConds](https://user-images.githubusercontent.com/48508691/118010699-947de900-b34f-11eb-8991-f233b2397570.png)
